go/vt/mysqlctl/metadata_tables.go

```diff
@@ -58,20 +59,44 @@ var (
 	}
 )
 
-// PopulateMetadataTables creates and fills the _vt.local_metadata table and
-// creates _vt.shard_metadata table. _vt.local_metadata table is
-// a per-tablet table that is never replicated. This allows queries
-// against local_metadata to return different values on different tablets,
-// which is used for communicating between Vitess and MySQL-level tools like
-// Orchestrator (https://github.com/openark/orchestrator).
+// MetadataManager manages the creation and filling of the _vt.local_metadata
+// and _vt.shard_metadata tables.
+type MetadataManager struct{}
+
+// CreateMetadataTables creates the metadata tables. See the package-level
+// function for more details.
+func (m *MetadataManager) CreateMetadataTables(mysqld MysqlDaemon, dbName string) error {
+	return CreateMetadataTables(mysqld, dbName)
+}
+
+// PopulateMetadataTables creates and fills the metadata tables. See the
+// package-level function for more details.
+func (m *MetadataManager) PopulateMetadataTables(mysqld MysqlDaemon, localMetadata map[string]string, dbName string) error {
+	return PopulateMetadataTables(mysqld, localMetadata, dbName)
+}
+
+// UpsertLocalMetadata adds the given metadata map to the _vt.local_metadata
+// table, updating any duplicate rows to the values in the map. See the package-
+// level function for more details.
+func (m *MetadataManager) UpsertLocalMetadata(mysqld MysqlDaemon, localMetadata map[string]string, dbName string) error {
+	return UpsertLocalMetadata(mysqld, localMetadata, dbName)
+}
+
+// CreateMetadataTables creates the _vt.local_metadata and _vt.shard_metadata
+// tables.
+//
+// _vt.local_metadata table is a per-tablet table that is never replicated.
+// This allows queries against local_metadata to return different values on
+// different tablets, which is used for communicating between Vitess and
+// MySQL-level tools like Orchestrator (https://github.com/openark/orchestrator).
+//
 // _vt.shard_metadata is a replicated table with per-shard information, but it's
 // created here to make it easier to create it on databases that were running
 // old version of Vitess, or databases that are getting converted to run under
 // Vitess.
-func PopulateMetadataTables(mysqld MysqlDaemon, localMetadata map[string]string, dbName string) error {
-	log.Infof("Populating _vt.local_metadata table...")
+func CreateMetadataTables(mysqld MysqlDaemon, dbName string) error {
```

Comment:
It seems like this function is only called locally, rename it to `createMetadataTables()`?